script/license.txt

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2017 Jonathan Hall
```

Comment:
@flimzy can you also consider adding "Copyright (c) [yyyy] Kivik contributors"?

Reply:
I had the same thought earlier today. I'll be doing a massive licensing update.